src/internal/k8sconfig/go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/solarwinds/swi-k8s-opentelemetry-collector/internal/k8sconfig
+
+go 1.21.0
```

Comment:
We set version `1.22` in other go.mod files. Not that it makes much difference, anyway.

Reply:
Updated